ddtrace/llmobs/_integrations/anthropic.py

```diff
@@ -46,3 +142,11 @@ def record_usage(self, span: Span, usage: Dict[str, Any]) -> None:
 
         if input_tokens is not None and output_tokens is not None:
             span.set_metric("anthropic.response.usage.total_tokens", input_tokens + output_tokens)
+
+
+def _get_llmobs_metrics_tags(span):
+    return {
+        "input_tokens": span.get_metric("anthropic.response.usage.input_tokens"),
+        "output_tokens": span.get_metric("anthropic.response.usage.output_tokens"),
+        "total_tokens": span.get_metric("anthropic.response.usage.total_tokens"),
```

Comment:
Sorry, got notified by LLMObs backend that we're still using `prompt/completion_tokens` instead of `input/output_tokens`. Let's change the names accordingly.

Also, is `total_tokens` guaranteed to be present? If not, let's avoid submitting it and just submit `prompt/completion` token metrics.

Reply:
should we be changing the span tags on the APM integration or just for the MLObs integration?

Reply:
Total tokens should always be set if the message completion returned successfully

Reply:
Just for the MLObs integration, the APM integration is fine as is.